src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react';
+import { PeopleFilters } from '../PeopleFilters/PeopleFilters';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../../api';
+import { getFullInfo } from '../../helpers/getFullInfo';
+import { Person } from '../../types';
+import { ErrorAPI } from '../ErrorAPI/ErrorAPI';
+
+export const PeoplePage = () => {
+  const [hasError, setHasError] = useState<boolean>(false);
+  const [isLoadingData, setIsLoadingData] = useState<boolean>(true);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    setIsLoadingData(true);
```

Comment:
initial value is true and here you set true again, usually isLoading is false as init